handler/src/main/java/io/netty/handler/ssl/OpenSslCertificateException.java

```diff
@@ -24,6 +27,286 @@
  * SSL Record. This only work when {@link SslProvider#OPENSSL} is used.
  */
 public final class OpenSslCertificateException extends CertificateException {
+    public static final int X509_V_OK;
```

Comment:
The idea was to expose them to the user so the user will have these in one place which is not internal. We can also remove these and add them later if needed but my hope was that this will guide the users to not use the ones exposed by `CertificateVerifier`.